chord_metadata_service/restapi/utils.py

```diff
@@ -506,11 +486,11 @@ def get_range_stats(field_props):
     # Generate a list of When conditions that return a label for the given bin.
     # This is equivalent to an SQL CASE statement.
     whens = [When(
-                  **{f"{field}__gte": floor} if floor is not None else {},
-                  **{f"{field}__lt": ceil} if ceil is not None else {},
-                  then=Value(label)
-                 )
-             for floor, ceil, label in labelled_range_generator(field_props)]
+        **{f"{field}__gte": floor} if floor is not None else {},
+        **{f"{field}__lt": ceil} if ceil is not None else {},
+        then=Value(label)
+    )
+        for floor, ceil, label in labelled_range_generator(field_props)]
```

Comment:
slightly awkward formatting, maybe combine these 2 lines `) for floor, ...`

Reply:
My only contribution here is accidental whitespace changes (which VS Code seems to hide by default now? That's not a feature I wanted). 

Honestly, I'm expecting to refactor much of this file in the near future, if only to pull out the code that collects counts for "missing".